packages/kbn-proc-runner/proc.js

```diff
@@ -2,7 +2,7 @@ import { spawn } from 'child_process';
 import { statSync } from 'fs';
 
 import Rx from 'rxjs/Rx';
-import { brightBlack } from 'ansicolors';
+import { gray } from 'chalk';
```

Comment:
Gray is the new bright black. https://github.com/chalk/chalk/blob/master/readme.md

Reply:
I prefer `dim` personally 😉